Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift

```diff
@@ -514,7 +518,8 @@ struct CommunityView: View
         } catch {
             // TODO: we may be receiving decoding errors (or something else) based on reports in the dev chat
             // for now we will fail silently if the user has posts to view while we investigate further
-            assertionFailure("Unhandled error encountered, if you can reproduce this please raise a ticket/discuss in the dev chat")
+            print(String(describing: error))
+//            assertionFailure("Unhandled error encountered, if you can reproduce this please raise a ticket/discuss in the dev chat")
```

Comment:
can we put this assertion back in instead of the `print` please? 🙏

Reply:
done!